python/ray/serve/master.py

```diff
@@ -27,19 +30,30 @@ def __init__(self, kv_store_connector):
         self.route_table = RoutingTable(kv_store_connector)
         self.backend_table = BackendTable(kv_store_connector)
         self.policy_table = TrafficPolicyTable(kv_store_connector)
-        self.replica_tag_to_workers = dict()
+        # Dictionary of backend tag to dictionaries of replica tag to worker.
+        self.workers = defaultdict(dict)
 
         self.router = None
         self.http_proxy = None
         self.metric_monitor = None
 
+    async def _retry_actor_failures(self, call_actor_method):
```

Comment:
@stephanie-wang this is a workaround because we don't have at least once semantics for reconstructed actor methods (a use case for https://github.com/ray-project/ray/pull/7334).